django/library/models.py

```diff
@@ -2438,14 +2510,90 @@ def __str__(self):
         return f"[peer review] {invitation.candidate_reviewer} submitted? {self.reviewer_submitted}, recommendation: {self.get_recommendation_display()}"
 
 
-class CodeMeta:
+class CommonMetadata:
```

Comment:
it seems like this should be a collection of helper functions/static methods instead of a built dictionary, is this a worthwhile change?

Reply:
One of the initial goals was to also provide an easier way to mock out the metadata tests and improve testing speed by not needing a DB but the design didn't end up that way. That's part of the reason that the CodeMeta and DataCite metadata classes were initialized with a bare dictionary but without validation it's kind of a mess. Let's discuss at or after tomorrow's dev meeting?